src/data/markdown/docs/30 xk6-browser/01 Get started/03 Metrics.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Browser performance metrics'
+excerpt: 'An overview of the different browser performance metrics that xk6-browser tracks.'
+---
+
+Apart from the usual HTTP specific metrics that k6 already tracks, xk6-browser tracks the following browser specific performance metrics on top:
+
+- browser_dom_content_loaded - measures the time for when the page is loaded but scripts have just started to be executed.
```

Comment:
Can we use a table here to be consistent on how we represent other metric lists?

See example:
https://k6.io/docs/using-k6/metrics/#http-specific-built-in-metrics